back-end/src/app.module.ts

```diff
@@ -0,0 +1,25 @@
+import { Module } from '@nestjs/common';
+import { ImageModule } from './image-module/image.module';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { ImageEntity } from './image-module/image.entity';
+import { ArticleEntity } from './article-module/article.entity';
+import { CategoryEntity } from './category-module/category.entity';
+import { CategoryModule } from './category-module/category.module';
+import { ArticleModule } from './article-module/article.module';
+
+@Module({
+  imports: [ImageModule,CategoryModule,ArticleModule,
+    TypeOrmModule.forRoot({
+      type: 'postgres',
+      host:'localhost',
+      port:5432,
+      username: 'postgres',
+      password:'123456',
+      database:'abc_intern',
+      entities: [ImageEntity,ArticleEntity,CategoryEntity],
+      synchronize:true
+    })],
```

Comment:
mấy thông tin trong TypeOrmModule.forRoot({
      type: 'postgres',
      host:'localhost',
      port:5432,
      username: 'postgres',
      password:'123456',
      database:'abc_intern',
      entities: [ImageEntity,ArticleEntity,CategoryEntity],
      synchronize:true
    })], sau này mình làm trong mấy dự án nó là thông tin private nên anh thêm nó vô file .env nha, trong file back-end/.gitignore and đã thêm .env ở trỏng thì nó sẽ không đưa lên git